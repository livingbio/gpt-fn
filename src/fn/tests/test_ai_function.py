import pytest
from syrupy.assertion import SnapshotAssertion

from ..ai_function import ai_fn


@ai_fn
def fabnocci(n: int) -> int:  # type: ignore[empty-body]
    """return fabnocci number"""


@pytest.mark.vcr(filter_headers=["authorization"])
def test_ai_fn(snapshot: SnapshotAssertion) -> None:
    assert snapshot == fabnocci(10)
