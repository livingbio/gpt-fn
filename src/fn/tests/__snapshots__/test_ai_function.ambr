# serializer version: 1
# name: test_ai_fn
  55
# ---
